packages/commands/src/evmDeposit.ts

```
@@ -0,0 +1,125 @@
+import { Command } from "commander";
+import { ethers } from "ethers";
+import { networks } from "@zetachain/networks";
+import { type NetworksSchema } from "@zetachain/networks/dist/src/types";
+
+import { ZetaChainClient } from "../../client/src/client";
+import { evmDeposit } from "../../client/src/evmDeposit";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)